src/common/plugin/types.ts

```diff
@@ -61,11 +61,27 @@ export interface FeeRates {
   highFee: string
 }
 
+export const asFeeRatesCleaner = asObject({
+  lowFee: asString,
+  standardFeeLow: asString,
+  standardFeeHigh: asString,
+  highFee: asString
+})
+
 export interface SimpleFeeSettings extends FeeRates {
   standardFeeLowAmount: string
   standardFeeHighAmount: string
 }
 
+export const asSimpleFeeSettingsCleaner = asObject({
+  standardFeeLowAmount: asString,
+  standardFeeHighAmount: asString,
+  lowFee: asString,
+  standardFeeLow: asString,
+  standardFeeHigh: asString,
+  highFee: asString
+})
+
```

Comment:
nit: Avoid using "Cleaner" in the name of a cleaner. No need to have the word there when the "as" naming convention is enough to signal it's a cleaner. ;)